hazelcast/src/main/java/com/hazelcast/cache/impl/CacheDataSerializerHook.java

```diff
@@ -337,6 +349,36 @@ public IdentifiedDataSerializable createNew(Integer arg) {
                 return new CacheEntryIterationResult();
             }
         };
+        constructors[CACHE_PARTITION_LOST_EVENT_FILTER] = new ConstructorFunction<Integer, IdentifiedDataSerializable>() {
+            public IdentifiedDataSerializable createNew(Integer arg) {
+                return new CachePartitionLostEventFilter();
+            }
```

Comment:
Only used in member-side `CacheAddPartitionLostListenerMessageTask.call`, as far as I can tell this is not transferred over the network between client-member, only among members during member-side listener registration.
Client-side listener registration registers a member-side `InternalCachePartitionLostListenerAdapter` listener which then delivers `CachePartitionLostEvent` (a plain old `Serializable`) to the client.